src/cli/domain/package-server-script/bundle/config/external-dependencies-handlers.js

```diff
@@ -5,18 +5,21 @@
  * For more info http://webpack.github.io/docs/configuration.html#externals
  *
 */
+
 'use strict';
+
+const coreModules = require('builtin-modules');
 const format = require('stringformat');
 const _ = require('lodash');
+
 const strings = require('../../../../../resources');
 
 
 module.exports = function externalDependenciesHandlers(dependencies){
   const deps = dependencies || {};
 
-  const missingExternalDependecy = function(dep, dependencies) {
-    return !_.includes(_.keys(dependencies), dep);
-  };
+  const missingExternalDependency = (dep, dependencies) =>
+    !_.includes(_.keys(dependencies), dep) && !_.includes(coreModules, dep);
```

Comment:
This part is used for the CLI bundler. This is for making the bundler bypass the dependency in case is a core, by being treated in the same way normal deps are.